catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -21,10 +21,12 @@
            http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
            http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd">
 
-    <reference id="camelContext" interface="org.apache.camel.CamelContext"
+    <!-- Over blueprint timeout is changed from 5 minutes to 30 minutes (see pom.xml) -->
+    <!-- Set these 2 references back to the default 5 minutes  -->
+    <reference timeout="300000" id="camelContext" interface="org.apache.camel.CamelContext"
```

Comment:
I think these 2 references will be set back to 5 minutes based on the comment.

Reply:
Oops. I meant to leave this comment on the timeout in the pom, not the timeouts in the blueprint. Does my concern now make sense to you?

Reply:
❓ Does the timeout in the blueprint override the reference timeout? If the blueprint timeout is indefinite, and the reference timeout is 5 minutes, I think the blueprint timeout won't matter. I think we'd want to avoid any timeouts being indefinite in the CDM because the transformer wait timeout will only tell the transformer bootflag bundle to not publish the service bootflag service, meaning the CDM will wait forever if the transformer wait timeout is hit and the system will never fail install.

Reply:
The timeout in the pom (or it's default of 5 mins) overrides the reference timeouts defined in blueprint. The timeout would not be indefinite because the constructor times out via the `org.codice.ddf.platform.bootflag.transformerWaitTimeoutSeconds` (or its default if not specified). Essentially, we added a different way than the blueprint/reference to set the timeout. These different timeouts should not interfere with each other.

Reply:
I agree that we can't make it indefinite. I would think 30 minutes would be plenty long enough. I'd be surprised if a user would want to increase the value of `org.codice.ddf.platform.bootflag.transformerWaitTimeoutSeconds` to higher than 30 minutes. That's just my opinion. If you have a better suggestion for the value and why, I'd be happy to discuss.